product/outline.md

```diff
@@ -0,0 +1,55 @@
+# Audience
+
+While the content of this segment can be intersting to engineers in order to help their teams better navigate business decisions related to InnerSource the core audience are people in your team and across your organisation that deal with product development. The goal of this segment is to help you understand how InnerSource can help interdependent teams resolve requirements without syncing work to the point where both teams operate in lock-step.
+
+# Takeaways
+
+After going through this segment, you will:
+
+* understand how Agile software development relates to InnerSource:
+   * Which problems does InnerSource address that cause issues in typical agile teams? (Think "wait it out", "build workarounds" vs. "help with the solution".)
+   * In what aspects does InnerSource differ from agile or traditional software development - in particular when the same terminology and technology is used? (Think "Issue tracker for backlog/ board management vs. issue tracker for communication", "code review as quality gate vs. continuous code review as a conversation").
+*  understand how even with InnerSource the host team still has to plan time and capacity when receiving features as pull requests as opposed to features requests.
+*  understand additional negotiation possibilities that InnerSource brings (think "host teams implements tricky feature request of contributing team, in return the contributing team helps with simpler features that the host teams has higher up on their priority list")
+
```

Comment:
* Understand how agile and InnerSource complement each other
   * Well factored and well tested code leads to faster on-ramp for agile teams, but also helps InnerSource allowing for faster on-ramp for contributors
   * A focus on working together and not assigning individual tasks allows for easier on-ramp of contributors.
   * Existing local pairing can make cross team pairing and sharing of work in progress easier.
   * Existing test driven development means that confidence in contributions is higher.
   * XP teams have a "leave the code in better shape than you found it" attitude - mitigates the risk of decline in quality or co-hesion of code when faced with multiple contributions from different sources.